rollup.config.js

```diff
@@ -18,6 +18,9 @@ const configureRollupPlugins = (options = {}) =>
   [
     replace({
       'process.env.NODE_ENV': JSON.stringify('production'),
+      'process.env.npm_package_version': JSON.stringify(
+        process.env.npm_package_version
+      ),
```

Comment:
Nice. If this works I'll simplify app-kit again too again.

Reply:
But we don't use lerna here for bumping versions... I think that's why we can have this simpler

Reply:
![Screen Shot 2019-06-06 at 10 51 42 AM](https://user-images.githubusercontent.com/2425013/59019867-181c2a80-8849-11e9-8184-4d24e22eab2d.png)

screenshot of cjs file

Reply:
Nice.